src/components/Title.js

```diff
@@ -3,7 +3,7 @@ import { Heading, Box } from "@chakra-ui/core"
 
 export default ({ sub, main }) => (
     <Box textAlign="center">
-        <Heading color="blue" fontSize={["18px", "18px", "24px", "24px"]}>
+        <Heading color="blue.500" fontSize={["18px", "18px", "24px", "24px"]}>

```

Comment:
This shade of blue breaks accessibility.
See: https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1602550331584-57471.report.html#accessibility